evals/app-interface-generation.md

```diff
@@ -0,0 +1,89 @@
+   - Dashboard with KPIs
+   - Landing page with data grid
+   - Document management interface
+   - Comment/discussion interface
+
+2. Document current outputs focusing on:
+   - Visual sophistication level
+   - Use of professional styling patterns
+   - Component composition complexity
+   - Comparison to reference examples
+
+## Prompt Template
+
+- Run the following prompt for each test case
+- Run them in the same session if possible
+
+```
+## Goal
+
+Generate a high-quality, professional-level interface using Appian SAIL for the given test case
+
+## Business Context
+
+**Company:** Pacific Coast Credit Union
+**Background:** Regional credit union serving 85,000 members across California, modernizing member onboarding from paper-based processes to digital-first experience. Target: reduce processing time from 7 days to 24 hours while maintaining compliance.
+
+**Key Personas:**
+- **Carlos Martinez (New Member):** Small business owner needing quick account opening, limited time during business hours
+- **Maria Chen (Member Services Rep):** Processes 15-20 applications daily, needs visibility into completion status and compliance requirements
+- **Robert Kim (Compliance Officer):** BSA/AML specialist ensuring regulatory compliance, needs audit trails and risk screening
+
+## Specific Test Case
+[Single test case details]
+
+## Design System MCP Usage and Outputs
+
+- Use the design system MCP to generate an interface for each test case
+- When calling the MCP, start at the high level and work down based on what is needed for a given interface: layouts, patterns, then components
+- After thinking about how to generate each interface, review the sail-coding-guidance and accessibility checklist to avoid basic errors
+- IMPORTANT: Only use SAIL functions and syntax that are represented in the design system; don't make assumptions
+- Generate each interface in a separate .txt file
+- Don't include comments in the output file unless absolutely necessary
+```
+
+## Test Cases
+
+**1. KPI Dashboard**
+- **User Story:** As a member services manager, I want to see real-time application processing metrics, so that I can identify bottlenecks and allocate staff effectively.
+- **Sample Data:** Daily application volumes (150-200/month), processing times by stage, completion rates, staff performance metrics, compliance screening results
+- **Expected Enhancement:** layouts/dashboards.md architecture + patterns/key-performance-indicators + patterns/charts
+
+**2. Multi-step Application Wizard**
+- **User Story:** As a new member, I want to complete my account application in guided steps with progress tracking, so that I can apply outside business hours and save my progress.
+- **Sample Data:** Account types (Essential Checking, Business Advantage, Youth Savings), required documents by product type, identity verification steps, fee structures
+- **Expected Enhancement:** layouts/forms.md wizard patterns + patterns/document-cards + professional styling
```

Comment:
Wondering how useful "professional styling" is an expected enhancement, since we hope that all generated UIs apply professional styling - is this already handled by our "Evaluation Criteria" below?